### DISCUSSION TITLE
Help Question: Possible Dequeue CSS styles and JS for BS Contact Form 7 from every page

### DISCUSSION THREAD

**Discussion by hsankala** [bootscore/bootscore#593] [Q&A]
Hey Justin,

Just wondering if it's possible to dequeue the CSS style and JS script for this plugin so it doesn't pull down a copy of the style and script every page load and only load it on the contact page, for the performance gains!

- bs-cf7-style.min.css?ver=6.3.1
- bs-cf7-script.min.js?ver=6.3.1

Would it be possible to write a simple function like this below to achieve this (this is just a rough example of what I'm looking to do): 

```
// This is the page ID of my contact form
if ( is_page( '19630' ) ) {

      // enqueue the Bootstrap CF7 style and script
     wp_enqueue_style('main-cf7-bf-style', get_stylesheet_directory_uri() . '/css/bs-cf7-style.min.css', array('parent-style'), $modified_bootscoreChildCssTimeStamp);
    wp_enqueue_script('custom-js-cf-bf-script', get_stylesheet_directory_uri() . '/js/bs-cf7-script.min.js', false, '', true);
      
     }
     else
     {
        // If we're not on the contact form page don't load the styles/JS for Contact form 7 bootstrap form
        wp_dequeue_script('main-cf7-bf-style');
        remove_action( 'custom-js-cf-bf-script', 'bs-cf7_enqueue_scripts', 20 );
     }```

Thanks
Hefin

**Comment by crftwrk** [comment]
Think this won't work that way. You can grab the CSS/JS and enqueue them in your child if shortcode is present as described here https://www.wpexplorer.com/load-scripts-shortcode/.

But of course this plugin is no longer needed. However, once loaded, files gets cached and loaded on every page.
